conf/base.config

```diff
@@ -75,7 +75,7 @@ process {
         memory = { check_max( 50.GB * task.attempt, 'memory' ) }
     }
     withName: INTERPROSCAN {
-        cpus   = { check_max( 16    * task.attempt, 'cpus'   ) }
+        cpus   = { check_max( 32    * task.attempt, 'cpus'   ) }
```

Comment:
Interesting... how much faster is it? IPS runs hmmer internally and I know that using more that 4 cores for it tends to make it slower.

Reply:
I think I made this change, but never got to measure it

Reply:
I wasn't sure about this. I bumped this up from 16 to 32 because it was running excruciatingly slow and wasn't even completing within the 4 hour time limit which I had to bump too. Checked the documentation and they don't explicitly suggest not to go up to 32. I will reduce it back to 16 just in case.